.github/workflows/scripts/mass_erase_board.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#script accepts 2 arguments: device , device key in boards config
+
+#****************************************************************************************************
+# Function accepts parameters:device , CMSIS-DAP serial #
+function erase_with_openocd() {
```

Comment:
We could probably also erase the external flash devices with OpenOCD. The mass erase would take too long, but I think we can erase 1 or 2 sectors pretty quickly.

Reply:
Well maybe not :/ 

The MAX32655 doesn't have the XIP SPI, so the SPI erase won't work with OpenOCD.